commit 6a96323fb511565908a5a7ce7b1d6e0d40aa647d
Author: Lukas Slebodnik <lslebodn@redhat.com>
Date:   Mon Oct 24 10:03:32 2016 +0000

    BUILD: Accept krb5 1.15 for building the PAC plugin
    
    Reviewed-by: Sumit Bose <sbose@redhat.com>
    (cherry picked from commit 11d2a1183d7017f3d453d0a7046004b6968fefb5)

diff --git a/src/external/pac_responder.m4 b/src/external/pac_responder.m4
index 928b1d2..6b400d4 100644
--- a/src/external/pac_responder.m4
+++ b/src/external/pac_responder.m4
@@ -16,7 +16,8 @@ then
         Kerberos\ 5\ release\ 1.11* | \
         Kerberos\ 5\ release\ 1.12* | \
         Kerberos\ 5\ release\ 1.13* | \
-        Kerberos\ 5\ release\ 1.14*)
+        Kerberos\ 5\ release\ 1.14* | \
+        Kerberos\ 5\ release\ 1.15*)
             krb5_version_ok=yes
             AC_MSG_RESULT([yes])
             ;;
